docs/SETUP.md

```diff
@@ -1,119 +1,47 @@
-> Guide to setting up your very own ILP Kit.
-> Estimated time: a couple of hours.
+### ILP Kit installation
 
-> These instructions use **DigitalOcean's Node 6.9.1 image on Ubuntu**.
-> You can run ILP Kit on any other OS and hosting provider, but there
-> might be additional or different steps.
+Login to a hosted remote Linux server.
 
-> Alternative instructions for setup on **Google Cloud Platform** (Not Google AppEngine) are available here:
-> https://github.com/sappenin/ilpkit-google-cloud/tree/master/ilpkit-gce-single
-
-> _Don't hesitate to open issues or ask for help._
-
-### Table of Contents
-- [ILP Kit installation](#ilp-kit-setup)
-  - [Get a server instance](#get-a-server-instance)
-  - [Server setup](#server-setup)
-  - [Postgresql setup](#postgresql-setup)
-  - [ILP Kit setup](#ilp-kit-setup)
-- [Domain Setup](#domain-setup)
-  - [Subdomain setup](#subdomain-setup)
-  - [Set up SSL](#set-up-ssl)
-  - [Set up proxy for ILP Kit](#set-up-proxy-for-ilp-kit)
-- [ILP Kit configuration](#ilp-kit-configuration)
-  - [Configure environment](#configure-environment)
-  - [Launch ILP Kit](#launch-ilp-kit)
-  - [Issue some money](#issue-some-money)
-- [Enjoy](#enjoy)
-- [Appendix: Advanced Options](#appendix-advanced-options)
-  - [Mailgun setup](#mailgun-setup)
-  - [Github setup](#github-setup)
-  - [Issuing Money](#issuing-money)
-  - [systemd setup](#systemd-setup)
-
-# ILP Kit installation
-
-## Get a server instance
-
-### Buy server
-
-- On [DigitalOcean](https://cloud.digitalocean.com/droplets), click `Create Droplet`.
-- Select the option with 2 Gb of RAM (the $20 one).
-- For the operating system, go to `One-Click Apps` and select `Node 6.9.1`.
-- Select your ssh keys, and then confirm the purchase.
-
-I've named my server instance 'Niles', so that's the name I'll be using for the
-rest of the instructions.
-
-### Set up SSH host
-
-- Edit ~/.ssh/config
-- Add the lines:
-
-``` conf
-Host niles
-  Hostname 45.55.4.124 # IP from DigitalOcean
-  User root
-  Port 22
-```
-
-## Server setup
-
-### Log into your server
-
-``` sh
-$ ssh niles
-```
-
-### Install necessary packages
-
-``` sh
-root$ apt-get update && apt-get upgrade
-root$ apt-get install libssl-dev python build-essential libpq-dev postgresql postgresql-contrib
+``` 
+$ sudo apt-get update && sudo apt-get upgrade
+$ sudo apt-get install node libssl-dev python build-essential libpq-dev postgresql postgresql-contrib nginx letsencrypt
 ```
 
 Make sure your postgres version is at least 9.5 by running `postgres --version`.
 
 ### Add a non-root user
 
 ```
-# you don't want to stay logged in as root
-root$ adduser sharafian
-root$ usermod -aG sudo sharafian
+$ adduser sharafian
```

Comment:
This section is confusing, because the previous section already assumes a non-root user exists